docs/component-spec/modules.md

```diff
@@ -130,6 +128,12 @@ In the event of deprecating a module within Origami, the following steps must be
 3. Disable the Issues functionality from the deprecated module's repository.
 4. Update the repository's description to "deprecated - please use <module> instead" if it has been replaced and change the URL to point towards the replacement's repository on GitHub.
 
+#### Deprecation of a module feature
+
+* Deprecated code should go into a private deprecated file, or, if there’s an abundance of deprecated code, to a directory called `deprecated`. This way, it will be much easier to work with the new code while maintaining legacy code. It will also be easier to delete when making a major release. If separating deprecated code into its own `deprecated` file or directory is not pragmatic, modified code must be prepended and post-appended with a comment which begins ```@deprecated```, followed by text which describes the deprecation
+* Deprecated functions and mixins should log a warning stating that they are now deprecated and offering an alternative when there is one. This warning *may* also be added to the readme
+* An issue with label `breaking` must be created or updated to remind us to remove deprecated code upon a future major release
```

Comment:
I think the label that gets added automatically to the repos is `type: breaking`. Not sure whether we should match that here or not.